src/NServiceBus.NHibernate.Tests/ApprovalFiles/APIApprovals.Approve.approved.txt

```diff
@@ -10,6 +10,11 @@ namespace NServiceBus.Features
         public NHibernateGatewayDeduplication() { }
         protected override void Setup(NServiceBus.Features.FeatureConfigurationContext context) { }
     }
+    public class NHibernateOutbox : NServiceBus.Features.Feature
```

Comment:
Is this public just so it can be disabled?

Reply:
Nope. I tried to follow the convention here (this repo) which seems to be to make the feature public e.g. `NHibernateSagaStorage`. Honestly I am not sure what would be the consequences of disabling this feature via the feature API. 

But now that I think about, I guess this is following a bad pattern... I'll change it to internal.